src/main/java/thePackmaster/patches/contentcreatorpack/OnDrawCard.java

```diff
@@ -0,0 +1,34 @@
+package thePackmaster.patches.contentcreatorpack;
+
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import javassist.CtBehavior;
+
+public class OnDrawCard {
```

Comment:
Might be more accurate to call this `TrackCardsDrawnDuringTurnPatch`. I'm commenting on this most so that some future contributor searching for OnDraw is less likely to get confused -- with the current name this class sounds like a type of AbstractCard (one of the interfaces you would implement) and sounds like it would trigger for any draw at all.

Reply:
Agreed

Reply:
Done!